src/govuk/components/character-count/template.test.js

```diff
@@ -260,4 +260,23 @@ describe('Character count', () => {
       })
     })
   })
+
+  describe('no maximum', () => {
+    // If the template has no maxwords or maxlength to go for
+    // it needs to pass down any fallback hint to the JavaScript
+    // so it can inject the limit it may have received at instantiation
+    it('renders the fallback hint as a data attribute', () => {
+      const $ = render('character-count', examples['no maximum'])
+
+      // Fallback hint is passed as data attribute
+      const $component = $('[data-module]')
+      expect($component.attr('data-i18n.fallback-hint.other')).toEqual('No more than %{count} characters')
+
+      // A non-breaking space is set as the count message to reserve space
+      // and prevent a layout shift when the JavaScript component
+      // initialises and fills the element with the relevant text
+      const $countMessage = $('.govuk-character-count__message')
+      expect($countMessage.html()).toContain('&#xA0;')
```

Comment:
Worth rebasing against main – suspect this might fail once rebased on top of 7bbc33234ad431e88e526f1b698355022e68e74d.

Reply:
`main` is already rebased. Because this is a non breaking space, Cheerio still encodes it (though into its hexa decimal form rather than the initiaal `&nbsp;`).

Reply:
According to GitHub this branch is currently 26 commits behind main, including 7bbc33234ad431e88e526f1b698355022e68e74d.

But happy that it might not be an issue anyway 👍🏻

Reply:
Yup, I'd forgotten to push the rebasing, so got confused as to why things were still marked as OK on Github 🤦🏻